### PR TITLE
Move Project/Build GA Events

### PR DESCRIPTION
**Pull request by MDrakos**
https://www.pivotaltracker.com/story/show/174200192

### CODE REVIEW

File: internal/runners/activate/activate.go [hunk 1]
```diff
@@ -81,8 +60,6 @@ func (r *Activate) run(params *ActivateParams, activatorLoop activationLoopFunc)
 		}
 	}
 
-	go sendProjectIDToAnalytics(params.Namespace, filepath.Join(targetPath, constants.ConfigFileName))
```
Comment (MDrakos) [inline internal/runners/activate/activate.go:84]: Should we have another event for activation to replace this one?

Comment (Naatan) [reply internal/runners/activate/activate.go:84]: Martin is adding it in this PR:

https://github.com/ActiveState/cli/pull/857/files

File: internal/runners/activate/activate.go [hunk 2]
```diff
@@ -49,24 +46,6 @@ func (r *Activate) Run(params *ActivateParams) error {
...
-	platProject, fail := model.FetchProjectByName(names.Owner, names.Project)
-	if fail != nil {
-		logging.Debug("error getting platform project: %v", fail.ToError())
-		return
-	}
-	projectID := platProject.ProjectID.String()
-	analytics.EventWithLabel(
-		analytics.CatBuild, analytics.ActBuildProject, projectID,
-	)
-}
```
Comment (Naatan) [inline internal/runners/activate/activate.go:68]: Leave the function and just remove the event, Martin is going to need this function for his PR.

Comment (MDrakos) [reply internal/runners/activate/activate.go:68]: Not addressing this as per slack discussion.

File: pkg/platform/runtime/download.go [hunk 3]
```diff
@@ -195,7 +184,9 @@ func (r *Download) FetchArtifacts() (*FetchArtifactsResult, *failures.Failure) {
 
 		case <-buildStatus.Started:
 			logging.Debug("BuildStarted")
-			sendProjectIDToAnalytics(r.owner, r.projectName)
+			analytics.EventWithLabel(
+				analytics.CatBuild, analytics.ActBuildProject, fmt.Sprintf("%s/%s", r.owner, r.projectName),
```
Comment (Naatan) [inline pkg/platform/runtime/download.go:188]: Please use http://github.com/ActiveState/cli/blob/75e69a2f7f6f60d024f4b51380abcb40ab0375aa/pkg/project/namespace.go#L41-L41

**Pull request merged**